src/pyhf/workspace.py

```diff
@@ -685,6 +698,7 @@ def combine(cls, left, right, join='none'):
             left (~pyhf.workspace.Workspace): A workspace
             right (~pyhf.workspace.Workspace): Another workspace
             join (:obj:`str`): How to join the two workspaces. Pick from "none", "outer", "left outer", or "right outer".
+            merge_channels (:obj:`bool`): Whether or not to merge channels when performing the combine. This is only done with "left outer" or "right outer" options.
```

Comment:
Above you have ```merge (`bool`)``` and here ``` merge_channels (:obj:`bool`)```. It would be good if we can just stick to one style.

Reply:
they're different functions. one is `_join_channels` and this one is the public-facing API.

Reply:
I mean the syntax of ````bool```` vs ``` :obj:`bool` ```

Reply:
#1116

Reply:
This is now https://github.com/scikit-hep/pyhf/issues/1116